docs/index_whyte.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: page
+---
+# Build a User Task Management System with *To Do*
+
+Connect our database to the apps your customers use every day
```

Comment:
This is pretty generic.
Could you try a more specific value proposition that highlights a product feature?